src/include/ca-management/new_cert_read_write.rb

```diff
@@ -1932,7 +1933,7 @@ def create_default_CA_certificate
             _(
               "The default certificate can also be created in\nthe CA Management module.\n"
             )
-          )
+          ) unless Mode.autoinst
```

Comment:
The module will not be called by autoupgrade. But it does not hurt if we are taking both to be sure.